### ISSUE TITLE
Changes needed in Add Profile Activity [A11y]

### ISSUE THREAD

**Issue by Arjupta** [oppia/oppia-android#2258]
**Describe the bug**

Add Profile Activity is not fully compatible with some users from the Accessibility point of view.

![Screenshot_2020-12-06-23-20-28-91_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102060207-e4e03180-3e17-11eb-957a-25bfae79ef91.jpg)

Issue 1 - The circular image view, edit image button does and info button do not have any content label.
Issue 2 - The Profile Input Views present for Entering
 - Name of the user
 - 3-digit Pin
 - Confirming that pin 
 
Does not have a content label

Issue 3 - Certain Views does not have the correct contrast ratios (Inactive state) as mentioned in the screenshots. To solve this one must find a way to make the views disabled instead of being in the non-clickable state only

**To Reproduce**
Steps to reproduce the behaviour:
1. Start the Accessibility Scanner App on your device
2. Open the Oppia Android App and try to add a new user. 
3. Scan the screenshots with the Accessibility Scanner
4. You will see the tests that are not passed by that screen
5. Repeat this for other screens

**Expected behaviour**
All the tests must be passed by scanning the screenshots for both Tablet and Mobile Devices in every orientation.

**Screenshots**

| Issue 1  |    Issue 2    | Issue 3 |
|:--------:|:-------------:|:-:|
|![Screenshot_2020-12-06-23-19-56-73_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102063912-a600aa80-3e1c-11eb-8df0-0b2a9e8e49cf.jpg)|  ![Screenshot_2020-12-06-23-20-07-56_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102064867-d5fc7d80-3e1d-11eb-85a8-b003646aaa50.jpg) | ![Screenshot_2020-12-06-23-22-31-88_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102066072-62f40680-3e1f-11eb-9a27-be9d51461f2a.jpg) | 
|![Screenshot_2020-12-06-23-20-02-50_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102064127-f0822700-3e1c-11eb-9b6a-791440176351.jpg)| ![Screenshot_2020-12-06-23-22-10-00_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102064929-e876b700-3e1d-11eb-8883-60dc0de48b25.jpg) | ![Screenshot_2020-12-06-23-22-25-24_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102066076-638c9d00-3e1f-11eb-9ec6-54cb2de4a0ee.jpg) | 
|![Screenshot_2020-12-06-23-20-11-66_80a9469f53bfe75dbc6e8989be28e2f0 (1)](https://user-images.githubusercontent.com/64064110/102066522-ee6d9780-3e1f-11eb-89fb-dcd5b9586844.jpg) | ![Screenshot_2020-12-06-23-22-16-36_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102064942-ea407a80-3e1d-11eb-941a-75202674233e.jpg)| ![Screenshot_2020-12-06-23-22-38-88_80a9469f53bfe75dbc6e8989be28e2f0](https://user-images.githubusercontent.com/64064110/102066090-68e9e780-3e1f-11eb-8943-1ec5225fe53b.jpg) | 

**Device**
- Real Device - RealMe3
- SDK version - 29

**Additional context**
The Screenshots are descriptive enough for understanding the issues but still, if there is a need of clarification then you can comment here. Multiple Pull Requests can also be made for solving these issues.

Reference - [https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner](https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner)

**Comment by yk1ax** [comment]
@ARJUPTA I would like to work on this issue. Can I get assigned ?

**Comment by Arjupta** [comment]
> @ARJUPTA I would like to work on this issue. Can I get assigned ?

You can work on this but your PR will merged only after our team members decide the correct labels, if we don't find your labels descriptive enough. 

Assigning this to you

**Comment by Arjupta** [comment]
### Checklist
 (Contributors can take one issue at a time)
* [ ]  `Issue-1` (not assigned to anyone)
* [ ]  `Issue-2` (not assigned to anyone)
* [ ]  `Issue-3` (assigned to @ARJUPTA)

**Comment by rt4914** [comment]
Opening because it was closed incorrectly because of github bug.
